Description: This is needed for crossbuilding packages
Author: Pascal Roeleven <dev@pascalroeleven.nl>

--- a/debian/control
+++ b/debian/control
@@ -23,7 +23,7 @@
   tzdata <!nocheck>,
   systemtap-sdt-dev [!hurd-amd64 !hurd-i386],
   valgrind-if-available,
-Build-Depends-Indep: python3-sphinx, python3-docs-theme, texinfo
+Build-Depends-Indep: python3-sphinx:all, python3-docs-theme:all, texinfo
 Standards-Version: 4.7.0
 Vcs-Browser: https://salsa.debian.org/cpython-team/python3
 Vcs-Git: https://salsa.debian.org/cpython-team/python3.git
--- a/debian/rules
+++ b/debian/rules
@@ -1452,7 +1452,7 @@
 	done
 
 	: # devhelp docs
-	cd $(buildd_static) && ./python ../debian/pyhtml2devhelp.py \
+	cd $(buildd_static) && python3.12 ../debian/pyhtml2devhelp.py \
 		../$(d_doc)/usr/share/doc/$(p_base)/html index.html $(VER) \
 		> ../$(d_doc)/usr/share/doc/$(p_base)/html/$(PVER).devhelp
 	gzip -9nv $(d_doc)/usr/share/doc/$(p_base)/html/$(PVER).devhelp
