Description: Re-enable ensurepip inside this package
Author: Pascal Roeleven <dev@pascalroeleven.nl>

--- a/debian/rules
+++ b/debian/rules
@@ -333,7 +333,7 @@
 	test_threading test_threading_local test_threadsignals \
 	test_concurrent_futures test_ctypes \
 	test_dbm_dumb test_dbm_ndbm test_pydoc test_sundry \
-	test_signal test_ioctl test_gdb test_ensurepip test_venv
+	test_signal test_ioctl test_gdb test_venv
 
 # TODO: these hang/fail with new OpenSSL
 PROFILE_EXCLUDES += \
@@ -370,7 +370,6 @@
 		--enable-loadable-sqlite-extensions \
 		--with-dbmliborder=$(dbmliborder) \
 		--with-computed-gotos \
-		--without-ensurepip \
 		--with-system-expat \
 		--with-ssl-default-suites=openssl \
 		--with-wheel-pkg-dir=/usr/share/python-wheels/ \
@@ -580,7 +579,7 @@
 endif
 TESTOPTS = -j 1 -w -u$(TEST_RESOURCES) --timeout=3600
 TEST_EXCLUDES =
-TEST_EXCLUDES += test_ensurepip test_venv
+TEST_EXCLUDES += test_venv
 ifeq ($(on_buildd),yes)
   TEST_EXCLUDES += test_tcl test_codecmaps_cn test_codecmaps_hk \
         test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw \
@@ -931,11 +930,6 @@
 	rm -f $(d)/$(scriptdir)/lib2to3/*.pickle
 	rm -f $(d)/usr/share/man/man1/python3.1
 
-	if [ -d $(d)/$(scriptdir)/ensurepip/_bundled ]; then \
-	  echo "don't ship the prebuilt ensurepip/_bundled wheels."; \
-	  rm -rf $(d)/$(scriptdir)/ensurepip/_bundled; \
-	fi
-
 	: # cannot build it, zlib maintainer won't provide a mingw build
 	find $(d) -name 'wininst*.exe' | xargs -r rm -f
 
--- a/Lib/ensurepip/__init__.py
+++ b/Lib/ensurepip/__init__.py
@@ -52,7 +52,7 @@
 # policies recommend against bundling dependencies. For example, Fedora
 # installs wheel packages in the /usr/share/python-wheels/ directory and don't
 # install the ensurepip._bundled package.
-_WHEEL_PKG_DIR = sysconfig.get_config_var('WHEEL_PKG_DIR')
+_WHEEL_PKG_DIR = sysconfig.get_config_var('DO_NOT_USE_WHEEL_PKG_DIR')
 
 
 def _find_packages(path):
--- a/debian/control
+++ b/debian/control
@@ -52,8 +52,6 @@
 Multi-Arch: allowed
 Priority: optional
 Depends: python3.12 (= ${binary:Version}),
- python3-pip-whl (>= 22.2),
- python3-setuptools-whl,
  ${shlibs:Depends}, ${misc:Depends},
 Breaks: python3-pip (<< 1.5.6-4)
 Description: Interactive high-level object-oriented language (pyvenv binary, version 3.12)
--- a/debian/control.in
+++ b/debian/control.in
@@ -52,8 +52,6 @@
 Multi-Arch: allowed
 Priority: @PRIO@
 Depends: @PVER@ (= ${binary:Version}),
- python3-pip-whl (>= 22.2),
- python3-setuptools-whl,
  ${shlibs:Depends}, ${misc:Depends},
 Breaks: python3-pip (<< 1.5.6-4)
 Description: Interactive high-level object-oriented language (pyvenv binary, version @VER@)
